css/blog.css

```diff
@@ -0,0 +1,238 @@
+:root {
+    --primary: #CFF4EC;
+    --primary-dark: #94D6C8;
+
+    --secondary: #E2F6CD;
+
+    --neutral: #F2F2F1;
+
+    --text-color: #2F2F2E; 
+
+    --btn-color: #4D7369;
+
+    --white: #ffffff;
+
+    --light-gray: #969696;
+}
+
+
+
+body{
+    box-sizing: border-box;
+    font-family: 'Quicksand', sans-serif;
+    color: var(--text-color);
+    margin: 0;
+    padding: 0;
+    overflow-X: hidden;
+}
+
+header {
+    padding-bottom: 20px;
+}
+
+h1, h3 {
+    font-family: 'Libre Baskerville', serif;
+    font-style: italic;
+    font-size: 1em;
+    line-height: 1.7em;
+    
+}
+
+h1, h2, h3 {
+    padding: 10px 0;
+}
+
+h2 {
+    font-family: 'Libre Baskerville', serif;
+    font-size: 2.6em;
+}
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: var(--text-color);
+}
+
+.margin-bottom {
+    margin-bottom: 25px;
+}
+
+.box-sizing {
+    padding: 80px 15px 40px 15px;
+}
+/* NAV */
+
+.nav-padding {
+    padding: 25px 25px;
+}
+
+/* APPOINTMENT BTN */
+
+.appointment-btn {
+    background-color: var(--primary);
+    border: none;
+    padding: 10px 20px;
+}
+
+.appointment-btn:hover {
+    background-color: var(--btn-color);
+}
+
+.display-md {
+    display: none;
+}
+
+/* MAIN */
+
+.content-center {
+    text-align: center;
+    
+}
+
+/* ARTICLES */
+
+.article-heading {
+    font-style: italic;
+    font-size: 1.4em;
+    line-height: 1.7em;
+    padding-top: 60px;
+    padding-bottom: 20px;
+    text-align: center;
+}
+
+.article-section {
+    background-color: var(--neutral);
+}
+
+.search-box {
+    background-color: var(--primary);
+    color: var(--white);
+}
+.input-box {
+    background: rgba(255, 255, 255, 0);
+    border: none;
+    padding-left: 10px;
+}
+
+input-box:focus{
+    display: none;
+}
+
+.blog-info {
+    margin-top: 40px;
+    margin-bottom: 20px;
+}
+
+.blogger-details>p {
+    font-size: 0.8em;
+    margin: 0 0 0 20px;
+}
+
+.fa-user:before{
+    background-color: var(--light-gray);
+    color: white;
+    border-radius: 50%;
+    padding: 10px;
+    margin: 0;
+}
+
+.article-details> h2 {
+    font-size: 1.9em
+}
+
+.article-box {
+    background-color: var(--white);
+    padding-bottom: 15px;
+}
+
+.article-btn {
+    background-color: var(--primary);
+    border: none;
+    margin-top: 30px;
+    height: 40px;
+}
+
+.article-btn:hover {
+    background-color: var(--btn-color);
+}
+
+/* CHAT ICON */
+
+
+.chat-icon {
+    color: var(--white);
+    background-color: var(--btn-color);
+    border-radius: 50%;
+    height: 40px;;
+    padding:  12px;
+    padding-right: 27px;
+}
+
+.chat-icon-fixed-padding {
+    padding: 20px 40px;;
+}
+/* FOOTER */
+
+.footer-section {
+    padding: 60px 0 80px 0;
```

Comment:
This is breaking your layout (it causes the white space on the right-hand side) because you're overriding the left- and right padding of the container. Only set the top and bottom padding here.